infrastructure/main.tf

```diff
@@ -1,3 +1,4 @@
 provider "azurerm" {
-  version = "1.44.0"
+  version = "=2.20.0"
```

Comment:
As discussed please check if we need to add empty perfest.tfvars . If the Answer is yes please add tfvars for all the environment as well, same checks need to be applied for fees-register-admin-web, bar-admin-web.

Reply:
I triggered build in perftest without perftest.tfvers file. It's working fine, hence I'm removing perftest.tfvers file from all PRS(pay bubble, fees-register-admin-web, bar-admin-web)

Reply:
PR looks good to me